backup.sh

```diff
@@ -95,6 +102,12 @@ function get_options {
 function get_localsettings_vars {
     LOCALSETTINGS="$INSTALL_DIR/LocalSettings.php"
 
+    if [ ! -e $LOCALSETTINGS ];then
```

Comment:
Does `$LOCALSETTINGS` need to be quoted here, in case it contains spaces?

Reply:
Should be fixed in lastest commit